src/io.ts

```diff
@@ -141,12 +139,15 @@ export async function createSourceData(config: ImageLayerConfig): Promise<Source
     }
 
     data = await loadMultiscales(node, attrs.multiscales);
-    axis_labels = getAxisLabelsFromMultiscales(attrs);
+    if (!config.axis_labels) {
+      // Update config axis_labels if present in multiscales
+      config.axis_labels = attrs.multiscales[0].axes;
```

Comment:
I think I'd rather just be explicit here for this edge case, rather than adding a util that might return undefined.